### ISSUE TITLE
Clearing Button no longer works

### ISSUE THREAD

**Issue by JkHowling** [ClanGenOfficial/clangen#1968]
Longtime lurker and player of dev mode, but first time bugreporting so hope the format is correct and helpful.

 **Type:** UI
**Summary:** Clearing button to view freshkill pile no longer functional (regardless of game_config settings regarding freshkill)
**Grade:** Important(?)
**Occurrence:** Permanent
**Reproduce:** ex.: Try opening freshkill pile after recent commit.
**Commit #:** e2970802b843ef537f1de33b855252c7662b923e

**Comment by Lixxis** [comment]
Can you check again if you are really on the development branch + the clan you play is in the expanded mode? 
The config files for prey only apply to clans which are not in classic mode. 

I tested it and can look into the clearing.

**Comment by JkHowling** [comment]
> Can you check again if you are really on the development branch + the clan you play is in the expanded mode? The config files for prey only apply to clans which are not in classic mode.
> 
> I tested it and can look into the clearing.

Yeah I'm 100% certain of both of those. It worked without any issues in the previous version, I updated, and the clearing label just reverted to a label. I can even open the clan in previous iterations and it works just fine. I tested updating and fresh downloads a couple times as well with little luck.

I did also noticed that updating removed the prey_config file from resources and put the settings found there into the game_config. I feel like that might've played a part? Because updating to the most recent dev build has returned the prey_config file, and now the clearing button once again works just fine.

**Comment by Lixxis** [comment]
> Yeah I'm 100% certain of both of those. It worked without any issues in the previous version, I updated, and the clearing label just reverted to a label. I can even open the clan in previous iterations and it works just fine. I tested updating and fresh downloads a couple times as well with little luck.
> 
> I did also noticed that updating removed the prey_config file from resources and put the settings found there into the game_config. I feel like that might've played a part? Because updating to the most recent dev build has returned the prey_config file, and now the clearing button once again works just fine.

So I'm a bit confused, is the newest version of development working for you or not?

If you updated the stable version (identified be the number in the bottom right corner 0.9.x) there is no prey system out. It is only available in the development branch of this repository. Maybe send a screenshot of the clan view to identify the problem.

**Comment by JkHowling** [comment]
Sorry, let me try to explain it a bit better.

**Commit #:** e2970802b843ef537f1de33b855252c7662b923e
Clearing button stopped working as soon as I updated to this dev build. The prey_config file that existed in builds prior to it vanished when I updated. The settings found there appeared at the top of my game_config file. Clearing button does not work on my expanded mode clan. Timeskipping still takes prey and still gives me "Clan doesn't have enough prey for next moon!" messages as well as starvation deaths.

**Commit #:** 4e610160714e0bf12a0daadaded493bffad12fc8
Clearing button now works again as normal upon updating to this build. The prey_config file has returned. Prey system works as normal. The issue is no longer occurring, as far as I can tell.

Here's the screenshot you asked for, not sure if it'll help at all but it does prove I'm on Dev, as I've said before. 
![image](https://github.com/ClanGenOfficial/clangen/assets/26156840/f82d46f1-fb8e-4664-aba1-65c3522d20f1)


Regardless, it seems to have resolved itself, so this can probably be closed. Sorry if this was a waste of time, I figured if it was occurring to me it might be happening to others as well and might be worth looking into.
